documentation/src/main/asciidoc/configuration.asciidoc

```diff
@@ -14,15 +14,15 @@ The TCK distribution comes with a directory
 `setup-examples` which contains two example projects
 for running the TCK. If you followed the instructions in <<installation>> you find the directory under
 `jakarta-bean-validation/tck/setup-examples`. Both setups are using
-Hibernate Validator as Jakarta Bean Validation Provider and Glassfish 5 as EE
+Hibernate Validator as Jakarta Bean Validation Provider and Eclipse GlassFish 5.1+ as EE
```

Comment:
as Jakarta EE container

Reply:
Pushed an additional commit to fix all the occurrences of EE.